content/english/anti-vegan/personal-choice.md

```diff
@@ -9,42 +9,48 @@ weight: 5
 
 ## Detail - Analytic
 
-The core idea underpinning the concept "It's my personal choice" is that we have the right to choose for ourselves the correct course of action and that the choice, once made, should be respected. 
+> A _golden rule_ of moral philosophy is; "all things being equal you ought to choose the action that does the least harm” 
+
+Anyone who accepts:
+
+1. It is not a necessary need to eat meat.
+2. Eating meat is the indirect cause of great harm.
+3. The golden rule that, "all things being equal you ought to choose the action that does the least harm”
 
-The hidden premises of respecting some-one else's choice go something like:
+will necessarily conclude:
 
-1. Everyone is allowed their own opinion even if you don't agree with it 
-2. My opinion informs my actions through choice
-3. My actions, choices, and opinions should all be afforded the same level of acceptance
+- Your _personal choice_ to eat meat causes unnecessary harm, does not warrant respect, and morally obligates me to speak out.
 
-This takes a freedom of thought/speech type model and applies it to actions under the guise of 'choice'. And it works intuitively because there are scenarios in which this makes sense!
+ 
+> Point 1 is a factual claim. If you doubt this claim please read [A well planned vegan diet is suitable for people of all ages.] ({{< ref "../facts/suitable-for-all" >}})
 
-- It's my choice to drop out of uni
-- It's my choice to dye my hair
-- It's my choice to spend all my money on a holiday instead of saving it for a house
+> Point 2 is the factual claim that animals are harmed by commercial farming and slaughter. See [Animal slaughter methods] ({{< ref "../facts/suitable-for-all" >}})
 
-The key thing that these scenarios have in common, and the necessary component for them to have the freedom of speech/thought model applied to them, is that their repercussions apply only to the person making the choice. Compare this with:
+> Point 3 is a *value statement*  that most people would accept.
 
-- It's my choice to set your house on fire
-- It's my choice to spend all your money on a holiday
-- It's my choice to kick your shins
+The conclusion holds that, where an individual could *easily choose otherwise* but instead *out of personal choice* selects an option that does uneccessary harm, there is no reason to respect that individual's choice and there is good reason to speak out against it.
 
-When the opinion "I think it would be a good thing to set your house on fire", which we may not agree with or even respect, is transformed into an action that affects someone other than the person that who holds that opinion, it loses its protection.  
+This holds equally true:
 
-You may think what you like, and do what you like, so long as it only affects you.
+- If you choose to litter
+- If you choose to buy cheap clothes while being fully aware they were made in a sweat shop.
```

Comment:
How about 
> If you choose to buy clothes made in a sweatshop when you have the means to buy ethically sourced clothes 
To avoid accidentally casting judgment on people on the poverty line.
Oh and one of these bullet points has a `.` and one does not. I don't mind which so long as they all match!

Reply:
Hmmm, 2nd hand etc? Is poverty really an excuse on this occasion. 'Ethically sourced' sounds like an expensive option.

If you choose to buy clothes knowing they were made in a sweat shop when you have other options.

Reply:
Full stops fixed

Reply:
`have other options` is even better!